src/configuration_env.js

```diff
@@ -21,18 +21,20 @@ const deprecatedEnvVars = {
   JAEGER_REPORTER_PASSWORD: 'JAEGER_PASSWORD',
   JAEGER_REPORTER_AGENT_HOST: 'JAEGER_AGENT_HOST',
   JAEGER_REPORTER_AGENT_PORT: 'JAEGER_AGENT_PORT',
-  JAEGER_DISABLE: 'JAEGER_DISABLED'
+  JAEGER_DISABLE: 'JAEGER_DISABLED',
 };
 
 export default class ConfigurationEnv {
   static _validateEnv() {
-    Object
-      .keys(deprecatedEnvVars)
-      .forEach(env => {
-        if (process.env[env]) {
-          console.warn(`You are using deprecated env variable ${env}. Use ${deprecatedEnvVars[env]} instead. \nDeprecated env variable will be removed in the next major release (4.x.x)`)
-        }
-      })
+    Object.keys(deprecatedEnvVars).forEach(env => {
+      if (process.env[env]) {
+        console.warn(
+          `You are using deprecated env variable ${env}. Use ${
+            deprecatedEnvVars[env]
+          } instead. \nDeprecated env variable will be removed in the next major release (4.x.x)`
```

Comment:
This feels like a very weird formatting of an extrapolated expression. Why do we need this change in this PR?

Reply:
Sorry for the confusion, I should have commented this change. This is happening when `prettier` [lints](https://github.com/jaegertracing/jaeger-client-node/blob/master/package.json#L81) the staged files. How would you like this to be resolved? Alter prettier config or accept new styling?

Reply:
To have a clean diff for now, I have overridden prettier and reverted these changes. Awaiting your final advice on the issue.